src/AddTodoForm.jsx

```diff
@@ -3,19 +3,23 @@ import { useState } from "react";
 import InputWithLabel from "./InputWithLabel";
 
 const AddTodoForm = (props) => {
-  const { onAddTodo } = props;
+  const { onAddTodo, onRemoveTodo, onPostData } = props;
   const [todoTitle, setTodoTitle] = useState("");
 
   const handleTitleChange = (event) => {
     const newTodoTitle = event.target.value;
     setTodoTitle(newTodoTitle);
   };
 
-  const handleAddTodo = (event) => {
+  const handleAddTodo = async (event) => {
     event.preventDefault();
-    console.log(todoTitle);
-    setTodoTitle("");
-    onAddTodo({ title: todoTitle, id: Date.now() });
+    try {
+      const newTodo = await onPostData(todoTitle);
+      onAddTodo({ title: newTodo.fields.title, id: newTodo.id });
+      setTodoTitle("");
```

Comment:
this is to reset the input field

```suggestion
      setTodoTitle("");
      event.target.reset();
```